client/my-sites/people/invite-people/index.jsx

```diff
@@ -582,8 +587,12 @@ class InvitePeople extends React.Component {
 
 	renderInviteLinkGenerateButton = () => {
 		const { translate } = this.props;
+		const classNames = classnames( 'invite-people__link-generate', {
+			'is-busy': this.state.isGeneratingInviteLinks,
+		} );
+
 		return (
-			<Button onClick={ this.generateInviteLinks } className="invite-people__link-generate">
+			<Button onClick={ this.generateInviteLinks } className={ classNames }>
```

Comment:
I think we're meant to use the `busy` prop on `Button` instead of adding the CSS class directly.

Reply:
Right, thanks for that!